Demo/Sources/Views/LongPressPlaybackView.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import PillarboxPlayer
+import SwiftUI
+
+#if os(iOS)
+private struct LongPressView<Content>: View where Content: View {
+    @GestureState private var isLongPressing = false
+    @State private var timer: Timer?
+
+    let minimumPressDuration: TimeInterval
+    @ViewBuilder let content: () -> Content
+    let action: (_ finished: Bool) -> Void
+
+    var body: some View {
+        content()
+            .simultaneousGesture(longPressGesture())
+            .onChange(of: isLongPressing) { isChanged in
+                timer?.invalidate()
+                if !isChanged {
+                    action(true)
+                } else {
+                    timer = Timer.scheduledTimer(withTimeInterval: minimumPressDuration, repeats: false) { _ in
+                        action(false)
+                    }
+                }
+            }
+    }
+
+    init(minimumPressDuration: TimeInterval = 1, content: @escaping () -> Content, action: @escaping (_ finished: Bool) -> Void) {
+        self.minimumPressDuration = minimumPressDuration
+        self.content = content
+        self.action = action
+    }
+
+    private func longPressGesture() -> some Gesture {
+        LongPressGesture(minimumDuration: .infinity, maximumDistance: .infinity)
+            .updating($isLongPressing) { value, state, transaction in
+                state = value
+                transaction.animation = .easeIn(duration: 2.0)
```

Comment:
Not seing where a 2-second animation takes place. Are animations properly forwarded? If an animation is desired, isn't 2 seconds a bit long?

Reply:
The animation is slightly perceptible during use. The switch between two speeds (current speed to 2x) seems to be smoother.

Reply:
With a duration of 2 seconds it should be clearly noticeable, though. I removed it and observed no noticeable change. Is the animation really there?